### PR TITLE
K8SPXC-1295 Update images and versions for 1.14.0 release

### PR DESCRIPTION
**Pull request by ptankov**
[K8SPXC-1295](https://jira.percona.com/browse/K8SPXC-1295) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
Update images and versions for 1.14.0 release

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?
- [ ] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PXC version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

[K8SPXC-1295]: https://perconadev.atlassian.net/browse/K8SPXC-1295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/cr-minimal.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
     schedule: "0 4 * * *"
   pxc:
     size: 1
-    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0
+    image: percona/percona-xtradb-cluster-operator:1.13.0
```
Comment (tplavcic) [inline deploy/cr-minimal.yaml:14]: This needs to be database image (`percona/percona-xtradb-cluster:8.0.35-27.1`), you have put operator image here.

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by tplavcic** [comment]
Please add a Jira ticket ID into PR title.

File: deploy/cr-minimal.yaml (hunk 1 above)
Comment (ptankov) [reply deploy/cr-minimal.yaml]: done

File: deploy/cw-operator.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: percona/percona-xtradb-cluster-operator:1.13.0
```
Comment (tplavcic) [inline deploy/cw-operator.yaml:45]: 1.14.0?

File: deploy/operator.yaml [hunk 3]
```diff
@@ -45,7 +45,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: percona/percona-xtradb-cluster-operator:1.13.0
```
Comment (tplavcic) [inline deploy/operator.yaml:48]: 1.14.0?

File: deploy/cw-bundle.yaml [hunk 4]
```diff
@@ -10648,7 +10648,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: percona/percona-xtradb-cluster-operator:1.13.0
```
Comment (tplavcic) [inline deploy/cw-bundle.yaml:10651]: 1.14.0?

File: deploy/cr.yaml [hunk 5]
```diff
@@ -561,7 +561,7 @@ spec:
         cpu: 300m
   backup:
 #    allowParallel: true
-    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+    image: percona/percona-xtradb-cluster-operator:1.13.0-pxc8.0-backup-pxb8.0.32
```
Comment (tplavcic) [inline deploy/cr.yaml:564]: 1.14.0?

File: deploy/cr.yaml [hunk 6]
```diff
@@ -533,7 +533,7 @@ spec:
 #     - 10.0.0.0/8
   logcollector:
     enabled: true
-    image: perconalab/percona-xtradb-cluster-operator:main-logcollector
+    image: percona/percona-xtradb-cluster-operator:1.13.0-logcollector
```
Comment (tplavcic) [inline deploy/cr.yaml:536]: 1.14.0?

File: deploy/cr.yaml [hunk 7]
```diff
@@ -371,7 +371,7 @@ spec:
   proxysql:
     enabled: false
     size: 3
-    image: perconalab/percona-xtradb-cluster-operator:main-proxysql
+    image: percona/percona-xtradb-cluster-operator:1.13.0-proxysql
```
Comment (tplavcic) [inline deploy/cr.yaml:374]: 1.14.0?

File: deploy/cr-minimal.yaml [hunk 8]
```diff
@@ -20,7 +20,7 @@ spec:
   haproxy:
     enabled: true
     size: 1
-    image: perconalab/percona-xtradb-cluster-operator:main-haproxy
+    image: percona/percona-xtradb-cluster-operator:1.13.0-haproxy
   logcollector:
     enabled: true
-    image: perconalab/percona-xtradb-cluster-operator:main-logcollector
+    image: percona/percona-xtradb-cluster-operator:1.13.0-logcollector
```
Comment (tplavcic) [inline deploy/cr-minimal.yaml:26]: 1.14.0?

File: deploy/cr-minimal.yaml [hunk 9]
```diff
@@ -20,7 +20,7 @@ spec:
   haproxy:
     enabled: true
     size: 1
-    image: perconalab/percona-xtradb-cluster-operator:main-haproxy
+    image: percona/percona-xtradb-cluster-operator:1.13.0-haproxy
```
Comment (tplavcic) [inline deploy/cr-minimal.yaml:23]: 1.14.0?

File: deploy/bundle.yaml [hunk 10]
```diff
@@ -10638,7 +10638,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: percona/percona-xtradb-cluster-operator:1.13.0
```
Comment (tplavcic) [inline deploy/bundle.yaml:10641]: 1.14.0?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: deploy/cr.yaml [hunk 11]
```diff
@@ -561,7 +561,7 @@ spec:
         cpu: 300m
   backup:
 #    allowParallel: true
-    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+    image: percona/percona-xtradb-cluster-operator:1.14.0-pxc8.0-backup-pxb8.0.32
```
Comment (tplavcic) [inline deploy/cr.yaml:564]: In PXC distribution we have XtraBackup `percona-xtrabackup-80-8.0.35-30.1.el9.x86_64.rpm` so we need to set the same version here so it should be `percona/percona-xtradb-cluster-operator:1.14.0-pxc8.0-backup-pxb8.0.35`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: deploy/cr.yaml (hunk 11 above)
Comment (ptankov) [reply deploy/cr.yaml]: done

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**